scipy/spatial/tests/test_kdtree.py

```diff
@@ -873,7 +873,7 @@ def test_kdtree_build_modes(kdtree_type):
 def test_kdtree_pickle(kdtree_type):
     # test if it is possible to pickle a KDTree
     try:
-        import cPickle as pickle
```

Comment:
Note for future reference: consider enabling the [`ignore_missing_imports`](https://mypy.readthedocs.io/en/stable/config_file.html#confval-ignore_missing_imports) option in the mypy ini file. This should silence any errors regarding third-party packages that lack any annotations whatsoever.